cmd/carbonapi/http/errors.go

```diff
@@ -0,0 +1,48 @@
+package http
+
+import (
+	"net"
+	"net/http"
+)
+
+type HttpError struct {
+	Method   string
+	Query    string
+	Err      error
+	HttpCode int
+}
+
+func HttpCode(err error, found bool) int {
+	if err == nil {
+		if !found {
+			return http.StatusNotFound
+		}
+		return http.StatusOK
+	}
+	if _, ok := err.(*net.OpError); ok {
```

Comment:
It's for low level error handling (from standard golang http client). No merry at this time. Yes, It can be wrapped. But it's encapsulated with some parent errors, like ErrFailedToFetch or ErrMaxTriesExceeded. So I can't easy extract merry error, but can extract original (root, not merry) error.